.github/ISSUE_TEMPLATE/question.md

```diff
@@ -1,26 +1,34 @@
 ---
 name: Question 🤔
 about: Usage question or discussion about Teleport.
-labels: "type: question"
+labels: question
 ---
-
 <!--
-  To make it easier for us to help you, please include as much useful information as possible.
+  Before opening a new issue, please search existing issues: https://github.com/gravitational/teleport/issues
 
   Useful Links:
-  - Documentation: https://gravitational.com/teleport/docs/
-
-  - If you're an OSS user and have a general question, please ask in our community forum: https://community.gravitational.com/c/teleport
+  - Documentation: https://goteleport.com/teleport/docs/
 
-  - For Pro and Enterprise users, please open a ticket via the Enterprise support portal: https://dashboard.gravitational.com/web/
-
-  Before opening a new issue, please search existing issues: https://github.com/gravitational/teleport/issues
+  - For Open Source users, please ask in our community forum: https://community.goteleport.com/c/teleport
+  - For Enterprise customers, please open a support ticket via the Enterprise support portal: https://dashboard.gravitational.com/web/
```

Comment:
Given that we're giving these two options, is there any point in having this issue type at all?

Reply:
In `config.yml`, perhaps we add a support option? Then eliminate `question.md`. I've only seen it used once. 

- Teleport Documentation
- Teleport Community Forum
- **Teleport Enterprise Support** -> https://dashboard.gravitational.com/web/
- Request a Demo

Reply:
Great idea!